myPortfolio.css

```diff
@@ -0,0 +1,119 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	font-size: 62.5%;
+}
+
+/* Header Section */
+
+.navigation {
+	background-color: #54c9c4;
+	display: flex;
+	justify-content: center;
+}
+
+nav a {
+	color: #2e2e2e;
+	text-decoration: none;
+	font-family: 'Kufam';
+	font-size: 2.5rem;
+	padding: 10px 40px;
+}
+
+nav a:hover {
+	color: #e1ac28;
+}
+
+header {
+	background-image: url("https://cdn.pixabay.com/photo/2020/04/27/17/58/petals-5100957_960_720.jpg");
+	background-size: cover;
+	height: 780px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: flex-end;
+
+}
+
+header::before {
+	content:'';
+	display: block;
+	background-color: rgba(84, 201, 196, 0.65);
+	width: 165px;
+	height: 165px;
+	border-radius: 50%;
+	position: relative;
+	top: 280px;
+}
+
+.overlap {
+	font-family: 'Kufam';
+	color: #54c9c4;
+}
+
+.myName {
+	font-size: 4rem;
+	color: #e1ac28;
+	position: relative;
+	top: 265px;
+	letter-spacing: 0.1rem;
+	padding-left: 4px;
+}
+
+p.overlap {
+	text-transform: uppercase;
+	font-size: 1.6rem;
+	letter-spacing: 1.2rem;
+	position: relative;
+	top: 230px;
+	padding-left: 22px;
+}
+
+.pseudoText {
+	font-family: 'Kufam';
+	font-size: 5.7rem;
+	font-weight: 600;
+	color: #2e2e2e;
+	position: relative;
+	bottom: 195px;
+	padding-left: 2px;
+}
+
+/* About Me Section */
+
+.sectionStyling {
+	display: flex;
+	font-family: 'Kufam';
+	background-color: #2e2e2e;
+	color: #f6f7f7;
+}
+
+.textLayout {
+	width: 40%;
+}
+
+.sectionHeading {
+	font-size: 2rem;
+}
+
+.sectionText {
+	font-size: 1.3rem;
+}
+
+.aboutMeImage {
+	width: 60%;
+}
+
+
+.portfolioImage {
+	width: 60%;
+}
+
+/*
+.textPortfolio {
```

Comment:
Couldn't see this in the HTMl, assuming it was replaced with textLayout?

Reply:
Yep, that's right. I  have deleted .textPortfolio now. Thanks